spec/lib/miq_automation_engine/service_methods/miq_ae_service_switch_spec.rb

```diff
@@ -0,0 +1,15 @@
+module MiqAeServiceSwitchSpec
+  describe MiqAeMethodService::MiqAeServiceSwitch do
+    it "#hosts" do
+      expect(described_class.instance_methods).to include(:hosts)
+    end
+
+    it "#guest_devices" do
+      expect(described_class.instance_methods).to include(:guest_devices)
+    end
+
+    it "#lans" do
+      expect(described_class.instance_methods).to include(:lans)
+    end
```

Comment:
@lfu While test that the service model exposes the associations we are expecting is good, the real check would be to make sure that the active_record object exposes the same associations.  To be effective that really should be done at a lower level and check all the defined service model objects.  Not something for this PR.